### PR TITLE
Fix labeling loop

### PR DESCRIPTION
**Pull request by corest**
Another round :/

### CODE REVIEW

File: v_4_5_0/files/conf/k8s-addons [hunk 1]
```diff
@@ -14,11 +14,16 @@ NAMESPACES="default kube-system"
 for namespace in ${NAMESPACES}
 do
     while
```
Comment (kopiczko) [inline v_4_5_0/files/conf/k8s-addons]: I think this should become `if`.

Comment (xh3b4sd) [reply v_4_5_0/files/conf/k8s-addons]: I think we should stop using bash magic where possible. 🙈

Comment (corest) [reply v_4_5_0/files/conf/k8s-addons]: It is while for the network glitches case, when `get namespaces` fails just because of api being unresponsive

Comment (kopiczko) [reply v_4_5_0/files/conf/k8s-addons]: Does it matter? `$?` will be non-zero anyway.

**Pull request merged**